app-ios/DroidKaigi2022/iOS/Info.plist

```diff
@@ -4,5 +4,10 @@
 <dict>
 	<key>ITSAppUsesNonExemptEncryption</key>
 	<false/>
+    <key>CFBundleLocalizations</key>
```

Comment:
This is required from the moko standpoint. It needs one place to read all of the supported languages.